benchmark.go

```diff
@@ -246,23 +257,33 @@ func runBenchmark(out output, logger *zap.Logger, allOpts Options, resolved reso
 		RPS:                rps,
 	}
 
+	var streamSummary *StreamSummary
+
+	if overall.totalStreamMessagesReceived > 0 || overall.totalStreamMessagesSent > 0 {
```

Comment:
Is there another way of checking that we are in a streaming mode?

Reply:
ATM no, we have to expose `encoding` from benchmarkCaller to check if we are benchmarking stream requests.

Reply:
I am fine with this, we can add  a method `Serializer` in the interface `benchmarkCaller`. What do you think?

Reply:
I added a method in `benchmarkCaller` to expose the type of underlying call (unary or streaming)